### PR TITLE
fix: Prevent Cucumber from choking on comments

### LINKED ISSUE #31: Replace named capture groups in Cucumber Regex
As of Cucumber-expression 5.0.14, named capture groups are no longer supported, due to it being buggy in Ruby.  This means that axe-matchers' Cucumber selector will not function with Cucumber 3.1.0. The regex will need to be replace. In the mean time people should stay on Cucumber 3.0.0.

### PR DESCRIPTION
**Pull request by WilcoFiers**
This fixes an issue which doesn't look like it ever worked... :(

### CODE REVIEW

File: lib/axe/cucumber/step.rb [hunk 1]
```diff
@@ -27,15 +27,15 @@ class Step
       # optionally specify ruleset via list of comma-separated tags
       (?-x:;? according to: (?<tags>.*?))?
 
-      # optionally specify rules (as comma-separated list of rule ids) to check
```
Comment (WilcoFiers) [inline lib/axe/cucumber/step.rb]: These brackets cause Cucumber to choke, Ruby passes them correctly, but Cucumber seems to do some preprocessing, which is where it fails, while our unit tests, which only test the regex itself still pass.

**Comment by WilcoFiers** [comment]
Still looking into the CI error. Stand by :)

**Comment by marcysutton** [comment]
Maybe you could disable the `region` rule since we're not likely to make changes to the test website

**Comment by jasonkarns** [comment]
Any further learning on this? I'm absolutely floored that cucumber would somehow be parsing the ruby file itself (would need to host its own ruby parser to do that) in order to parse the regex. (vs my understanding that cucumber just receives the regex as an object via the invocation of the `Then` method.)

**Pull request merged**